src/popup/components/Modals/AccountImport.vue

```diff
@@ -128,14 +132,24 @@ export default defineComponent({
         return;
       }
 
-      discovering.value = true;
-      setMnemonic(mnemonicParsed);
-      setBackedUpSeed(true);
-      await discoverAccounts();
-      props.resolve();
-      router.push(loginTargetLocation.value);
-      openEnableSecureLoginModal();
-      discovering.value = false;
+      try {
+        await setMnemonicAndInitializeAuthentication(mnemonicParsed);
+        discovering.value = true;
+        setBackedUpSeed(true);
+        await discoverAccounts();
+        props.resolve();
+        router.push(loginTargetLocation.value);
+
+        if (await checkBiometricLoginAvailability()) {
+          await openEnableBiometricLoginModal();
```

Comment:
Is no longer appearing. Instead you can see an biometric login modal for a second and then is disappears without user interaction.